src/Keycloak.Net.Core/Models/Root/OpenIdConnectConfig.cs

```diff
@@ -1,49 +1,49 @@
-﻿using Newtonsoft.Json;
+﻿namespace Keycloak.Net.Models.Root;
 
-namespace Keycloak.Net.Models.Root
+public class OpenIdConnectConfig
 {
-    public class OpenIdConnectConfig
-    {
-        [JsonProperty("userinfo.token.claim")]
-        public bool? UserInfoTokenClaim { get; set; }
+	[JsonPropertyName("userinfo.token.claim")]
+	public string? UserInfoTokenClaim { get; set; }
```

Comment:
I notice a few changes from `bool` to `string` in this file, what change was made to the response format that means that `bool` is no longer valid?

Reply:
For some reason in the latest version of Keycloak, some of these values were coming like `"userInfoTokenClaim": "true"` instead of `"userInfoTokenClaim": true`.
I didn't have the time to track at which Keycloak's version these fields changed, but at least with the latest one, it was this way and they were the reason a few tests were failing so I kept correcting the types to make them match.
Perhaps there's room for improvement by parsing the value and convert it to the actual type it should be, despite the API returning just a string.